deno/rest/v10/stageInstance.ts

```diff
@@ -24,6 +24,10 @@ export interface RESTPostAPIStageInstanceJSONBody {
 	 * Notify @everyone that a stage instance has started
 	 */
 	send_start_notification?: boolean;
+	/**
+	 * The guild scheduled event associated with this Stage instance
```

Comment:
Consistency with other descriptions:

```suggestion
	 * The guild scheduled event associated with this stage instance
```